jwt_proxy/api.py

```diff
@@ -54,6 +70,7 @@ def validate_jwt(relative_path):
     response_content = proxy_request(
         req=request,
         upstream_url=f"{current_app.config['UPSTREAM_SERVER']}/{relative_path}",
+        user_info=decoded_token.get("preferred_username"),
```

Comment:
Email should be slightly more consistent than Keycloak username
```suggestion
        user_info=decoded_token.get("email"),
```

Reply:
i didn't find "email" present in testing, could just be our test user lacks that attribute.
just pushed a change to prefer "email" if defined, or fall back to "preferred_username"